tests/unit/via/views/google_drive_test.py

```diff
@@ -12,6 +13,7 @@ def test_it_adds_headers(
     ):
         response = proxy_google_drive_file(pyramid_request)
 
+        assert response.status_code == 200
```

Comment:
Added a test here to make sure we don't return 204's all the time

Reply:
The test name `test_it_adds_headers()` might be inaccurate now. I don't think the status code is technically a header. Could be as simple as `test_status_and_headers()`